pyaerocom/web/aerocom_evaluation.py

```diff
@@ -885,6 +888,22 @@ def find_obs_matches(self, name_or_pattern):
                            .format(name_or_pattern))
         return matches
 
+    def _check_and_get_iface_names(self):
```

Comment:
I am not a fan of handling it this way. If I remember correctly `obs_config` entries are always instances of `ObsConfigEval`, correct? I think it could be safer and cleaner to handle this through a class method `get_web_iface_name` in `AerocomEvaluation`, e.g.:

```python
def get_web_iface_name(self, obs_name):
   try:
      name = self.obs_config[obs_name]['web_iface_name']
   except KeyError:
      name = obs_name
   if not isinstance(name, str):
      raise ValueError('Invalid value for web_iface_name in {}. Need str type'.format(name))
   return name

If you prefer to do it this way, then I suggest to make class attr. `iface_names` a property method, so that it is available on demand and does not require to be called by another method.
```

Reply:
I'm not immediately sure what other code changes would need to be done for this to work. Let me think about it.